proto/stream_service.proto

```diff
@@ -103,6 +103,21 @@ message WaitEpochCommitResponse {
   common.Status status = 1;
 }
 
+// Back pressure
+message GetBackPressureRequest {
+}
+
+message BackPressureInfo {
+    string actor_id = 1;          
+    string fragment_id = 2;   
+    string downstream_fragment_id = 3; 
```

Comment:
The data types can be integers. 

We use string in Prometheus because Prometheus require all labels to be string. But sicne this interface is particularly designed for back-pressire (as it's named `BackPressureInfo`), it's not that case.